upt_macports/tests/test_macports_package.py

```diff
@@ -96,6 +96,18 @@ def test_create_directories_permission_error(self, folder_name, m_mkdir):
             self.package._create_output_directories(self.package.upt_pkg,
                                                     '/ports/')
 
+    @mock.patch.object(MacPortsPackage, '_render_makefile_template',
+                       create=True, side_effect=PermissionError)
+    @mock.patch.object(MacPortsPackage, '_create_output_directories',
+                       create=True)
+    @mock.patch.object(MacPortsPackage, '_create_portfile',
+                       create=True)
+    def test_render_makefile_error(self, render, dir, portfile):
```

Comment:
Isn't the order of the arguments reversed?

I think it should be:

```python
@mock1
@mock2
def test_foo(self, mock2, mock1):
```

Also, is "create=True" really needed here?